WordPress/src/main/java/org/wordpress/android/ui/accounts/login/LoginEpilogueFragment.java

```diff
@@ -92,17 +98,30 @@ protected void setupLabel(@NonNull TextView label) {
 
     @Override
     protected ViewGroup createMainView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
-        return (ViewGroup) inflater.inflate(R.layout.login_epilogue_screen, container, false);
+        return (ViewGroup) inflater.inflate(loginEpilogueScreenResource(), container, false);
+    }
+
+    @LayoutRes
+    private int loginEpilogueScreenResource() {
+        if (mOnboardingImprovementsFeatureConfig.isEnabled()) {
+            if (mAdapter.getBlogsForCurrentView().size() <= EXPANDED_UI_THRESHOLD) {
+                return R.layout.login_epilogue_screen_new;
+            } else {
+                return R.layout.login_epilogue_screen_new_expanded;
+            }
+        } else {
+            return R.layout.login_epilogue_screen;
+        }
```

Comment:
Warning ⚠️:  IIRC we discussed not making any changes to the epilogue screen for the Jetpack app or at least not show the create site option. I'd suggest either we update the condition for the appropriate layout display in this PR or add a note in the task to not forget about it.

Reply:
👋 @ashiagr !

Great observation, I added a additional todo task on the parent issue [here](https://github.com/wordpress-mobile/WordPress-Android/issues/14990), which will deal with everything `Jetpack App` related. I think it is better for us to dedicate some additional time, add more thought and testing to when it comes to the Jetpack app. This way, it will also make this PR less complex and have it focusing on the WordPress app alone, at least until the full solution is complete. Wdyt?

Reply:
Works for me 🙂, thanks for adding the additional todo task 🙏.